bin/check_samplesheet.py

```diff
@@ -180,7 +199,21 @@ def check_samplesheet(file_in, file_out):
     ## Write validated samplesheet with appropriate columns
     if len(sample_mapping_dict) > 0:
         with open(file_out, "w") as fout:
-            fout.write(",".join(["sample", "single_end", "fastq_1", "fastq_2", "expected_cells", "seq_center"]) + "\n")
+            fout.write(
+                ",".join(
+                    [
+                        "sample",
+                        "single_end",
+                        "fastq_1",
+                        "fastq_2",
+                        "expected_cells",
+                        "seq_center",
+                        "fastq_barcode",
```

Comment:
What's the `fastq_barcode`? Does ATAC come with 3 fastq files per sample?

Reply:
I just quote from (https://support.10xgenomics.com/single-cell-multiome-atac-gex/software/pipelines/latest/using/fastq-input) for the file definition of the scATAC files of the 10x multiome scATAC + scRNA-seq

Where Read Type is one of:

I1: Dual index i7 read (optional)
R1: Read 1
**R2: Dual index i5 read** --> fastq_barcode
R3: Read 2
Alternatively, Cell Ranger ARC will also accept ATAC FASTQs in this format:

I1: Dual index i7 read (optional)
R1: Read 1
**I2: Dual index i5 read** --> fastqc_barcode
R2: Read 2